README.md

```diff
@@ -1,64 +1,109 @@
 
-In a terminal, clone the project with:
+## Features
 
-    $ git clone https://github.com/balena-io/open-balena.git
+- **Simple provisioning**: Adding devices to your fleet is a breeze
+- **Easy updates**: Remotely update the software on your devices with a single command
+- **Container-based**: Benefit from the power of virtualization, optimized for the edge
+- **Scalable**: Deploy and manage one device, or one million
+- **Powerful API & SDK**: Extend openBalena to fit your needs
+- **Built-in VPN**: Access your devices regardless of their network environment
 
-Change into the `open-balena` directory and run the configuration script.
-This will create a new directory, `config`, and generate appropriate SSL
-certificates and configuration for the instance.
 
-    $ ./scripts/quickstart
+## Roadmap
 
-You may optionally configure the instance to run under a custom domain name.
-The default is `openbalena.local`. For example:
+openBalena is currently in beta. While fully functional, it lacks features we
+consider important before we can comfortably call it production-ready. During
+this phase, also expect things to break. The following functionality is planned:
 
-    $ ./scripts/quickstart -d mydomain.com
+- Full documentation
+- Full test suite
+- Simple deployment
+- Remote host OS updates
+- Support for custom device types
 
-For more available options, see the script's help:
 
-    $ ./scripts/quickstart -h
+## Contributing
 
-Start the instance with:
+Everyone is welcome to contribute to openBalena. There are many different ways
+to get involved apart from submitting pull requests, including helping other
+users on the [forums][forums], reporting or triaging [issues][issue-tracker],
+reviewing and discussing [pull requests][pulls], or just spreading the word.
 
-    $ ./scripts/compose up -d
+All of openBalena is hosted on GitHub. Apart from its constituent components
+([API][open-balena-api], [VPN][open-balena-vpn], [Registry][open-balena-registry],
+[S3][open-balena-s3], [DB][open-balena-db]), contributions are also welcome to
+its client-side software such as the [balena CLI][balena-cli], the [balena SDK][balena-sdk],
+[balenaOS][balena-os] and [balenaEngine][balena-engine].
 
-Stop the instance with:
 
-    $ ./scripts/compose stop
+## Getting Started
 
-To remove all traces of the instance, run the following commands and finally
-delete the configuration folder.
+Our [Getting Started Guide][getting-started] gives you the most direct path to
+a working openBalena installation and successful deployment of your first project
+to your device.
 
-**WARNING**: This will remove *all* data.
 
-    $ ./scripts/compose kill
-    $ ./scripts/compose down
-    $ docker volume rm openbalena_s3 openbalena_redis openbalena_db openbalena_registry
+## Documentation
 
-### macOS & Windows
+While we're working on the project documentation, please refer to the
+[balenaCloud documentation][documentation]. BalenaCloud is built on top of
```

Comment:
```
[balenaCloud documentation][documentation]. BalenaCloud is built on top of
```
Looks like editor has auto-capitalised `BalenaCloud`?

Reply:
Additionally, this seems to link to https://www.balena.io/docs/learn/ which doesn't exist (https://www.balena.io/docs/learn/welcome/introduction/ does).

Reply:
I think that capitalization is actually correct, we do capitalize balena at the start of sentences.